File: fbi_web/src/com/fr/bi/web/js/data/filter/condition/empty.filter.js
;
!(function(){
    BI.EmptyFilter = function(){

    };
    BI.EmptyFilter.prototype = {
        constructor: BI.EmptyFilter,

        isQualified: function(){
            return false;
        },

        getFilterResult: function() {
            return [];
        }
    }
})();

File: fbi_engine_analyse/src/com/fr/bi/cal/analyze/report/report/widget/chart/newstyle/filter/condition/EmptyFilter.java
package com.fr.bi.cal.analyze.report.report.widget.chart.newstyle.filter.condition;

import com.fr.bi.cal.analyze.report.report.widget.chart.newstyle.filter.objectcondition.AbstractXYZFilter;
import com.fr.json.JSONArray;
import com.fr.json.JSONException;

/**
 * empty condition
 * Created by AstronautOO7 on 2016/9/19.
 */
public class EmptyFilter extends AbstractXYZFilter {

    @Override
    public JSONArray getFilterResult(JSONArray array) throws JSONException {
        return new JSONArray();
    }
}
